samples/broker/dlq/README.md

```diff
@@ -206,41 +206,45 @@ EOF
 ### Create Failer
 
 ```sh
-ko apply -n dlq-demo -f ./config/failer
+kubectl apply -f 'https://storage.googleapis.com/knative-nightly/eventing-rabbitmq/latest/failer.yaml'
 ```
 
 ### Check the results
 
-Look at the failer pod logs, you see it's receiving both 200/500 responses.
+Look at the failer pod logs, you see it's receiving both 200/503 responses.
 
 ```sh
 vaikas-a01:wabbit vaikas$ kubectl -n dlq-demo -l='serving.knative.dev/service=failer' logs -c user-container
-2020/10/02 11:35:00 using response code: 500
-2020/10/02 11:35:00 using response code: 200
-2020/10/02 11:36:00 using response code: 200
-2020/10/02 11:36:00 using response code: 500
-2020/10/02 11:37:00 using response code: 200
-2020/10/02 11:37:00 using response code: 500
-2020/10/02 11:38:00 using response code: 200
-2020/10/02 11:38:00 using response code: 500
-2020/10/02 11:39:00 using response code: 500
+2020/10/06 10:35:00 using response code: 200
+2020/10/06 10:35:00 using response code: 503
+2020/10/06 10:35:00 using response code: 503
+2020/10/06 10:35:00 using response code: 503
+2020/10/06 10:35:01 using response code: 503
+2020/10/06 10:35:01 using response code: 503
+2020/10/06 10:35:03 using response code: 503
 ```
 
-You see there are both 200 / 500 events there.
+You see there are both 200 / 503 events there. And more importantly, you can see
+that 200 is only sent once to the failer since it's processed
+correctly. However, the 503 is sent a total of 6 times because we have specified
+the retry of 5 (original, plus 5 retries for a total of 6 log entries).
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
that 200 is only sent once to the failer since it's processed correctly.
However, the 503 is sent a total of 6 times because we have specified the retry
of 5 (original, plus 5 retries for a total of 6 log entries).
```